File: 30-days-of-code/day6/Solution.java
import java.util.*;
import java.io.*;

public class Solution {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int testCount = sc.nextInt();

        for (int t = 0; t < testCount; t++) {
            String S = sc.next();
            String oddLetters = "";
            String evenLetters = "";

            for (int i = 0; i < S.length(); i++) {
                if (i % 2 == 0)
                    evenLetters += S.charAt(i);
                else
                    oddLetters += S.charAt(i);
            }

            System.out.println(evenLetters + " " + oddLetters);
        }
    }
}

File: 30-days-of-code/day6/solution.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{
    int test_count;
    cin >> test_count;

    while (test_count--)
    {
        string S;
        cin >> S;

        string odd_letters = "";
        string even_letters = "";

        for (int i = 0; i < S.size(); i++)
        {
            if (i % 2 == 0)
                even_letters += S[i];
            else
                odd_letters += S[i];
        }

        cout << even_letters << " " << odd_letters << "\n";
    }
}